lib/multi.c

```diff
@@ -374,6 +374,11 @@ struct Curl_multi *Curl_multi_handle(int hashsize, /* socket hash */
   multi->max_concurrent_streams = 100;
   multi->ipv6_works = Curl_ipv6works(NULL);
 
+#ifdef USE_WINSOCK
+  multi->wsa_event = WSACreateEvent();
+  if(!multi->wsa_event)
```

Comment:
I just checked that WSACreateEvent returns WSA_INVALID_EVENT on failure. Please check for that value instead.

I will also have to update my sockfilt.c code of course.

Reply:
`#define WSA_INVALID_EVENT       ((WSAEVENT)NULL)`
I can change the code to use the macro if you like, but it's the same check either way.

Reply:
Yes, that makes it "future-proof" in case Microsoft changes something.